Ensure actions have name and description translations

epenet opened the pull request:
## Proposed change
<!--
  Describe the big picture of your changes here to communicate to the
  maintainers why we should accept this pull request. If it fixes a bug
  or resolves a feature request, be sure to link to that issue in the
  additional information section.
-->
Follow-up to #158225

## Type of change
<!--
  What type of change does your PR introduce to Home Assistant?
  NOTE: Please, check only 1! box!
  If your PR requires multiple boxes to be checked, you'll most likely need to
  split it into multiple PRs. This makes things easier and faster to code review.
-->

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New integration (thank you!)
- [ ] New feature (which adds functionality to an existing integration)
- [ ] Deprecation (breaking change to happen in the future)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [x] Code quality improvements to existing code or addition of tests

## Additional information
<!--
  Details are important, and help maintainers processing your PR.
  Please be sure to fill out additional details, if applicable.
-->

- This PR fixes or closes issue: fixes #
- This PR is related to issue: 
- Link to documentation pull request: 
- Link to developer documentation pull request: 
- Link to frontend pull request: 

## Checklist
<!--
  Put an `x` in the boxes that apply. You can also fill these out after
  creating the PR. If you're unsure about any of them, don't hesitate to ask.
  We're here to help! This is simply a reminder of what we are going to look
  for before merging your code.

  AI tools are welcome, but contributors are responsible for *fully*
  understanding the code before submitting a PR.
-->

- [ ] I understand the code I am submitting and can explain how it works.
- [ ] The code change is tested and works locally.
- [ ] Local tests pass. **Your PR cannot be merged unless tests pass**
- [ ] There is no commented out code in this PR.
- [ ] I have followed the [development checklist][dev-checklist]
- [ ] I have followed the [perfect PR recommendations][perfect-pr]
- [ ] The code has been formatted using Ruff (`ruff format homeassistant tests`)
- [ ] Tests have been added to verify that the new code works.
- [ ] Any generated code has been carefully reviewed for correctness and compliance with project standards.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

If the code communicates with devices, web services, or third-party tools:

- [ ] The [manifest file][manifest-docs] has all fields filled out correctly.  
      Updated and included derived files by running: `python3 -m script.hassfest`.
- [ ] New or updated dependencies have been added to `requirements_all.txt`.  
      Updated by running `python3 -m script.gen_requirements_all`.
- [ ] For the updated dependencies - a link to the changelog, or at minimum a diff between library versions is added to the PR description.

<!--
  This project is very active and we have a high turnover of pull requests.

  Unfortunately, the number of incoming pull requests is higher than what our
  reviewers can review and merge so there is a long backlog of pull requests
  waiting for review. You can help here!
  
  By reviewing another pull request, you will help raise the code quality of
  that pull request and the final review will be faster. This way the general
  pace of pull request reviews will go up and your wait time will go down.
  
  When picking a pull request to review, try to choose one that hasn't yet
  been reviewed.

  Thanks for helping out!
-->

To help with the load of incoming pull requests:

- [ ] I have reviewed two other [open pull requests][prs] in this repository.

[prs]: https://github.com/home-assistant/core/pulls?q=is%3Aopen+is%3Apr+-author%3A%40me+-draft%3Atrue+-label%3Awaiting-for-upstream+sort%3Acreated-desc+review%3Anone+-status%3Afailure

<!--
  Thank you for contributing <3

  Below, some useful links you could explore:
-->
[dev-checklist]: https://developers.home-assistant.io/docs/development_checklist/
[manifest-docs]: https://developers.home-assistant.io/docs/creating_integration_manifest/
[quality-scale]: https://developers.home-assistant.io/docs/integration_quality_scale_index/
[docs-repository]: https://github.com/home-assistant/home-assistant.io
[perfect-pr]: https://developers.home-assistant.io/docs/review-process/#creating-the-perfect-pr

arturpragacz reviewed (commented):
For `onkyo` can you instead patch out `async_setup_services` in `onkyo/__init__py`? We need the service in just a single test for the media player, so that solution would be way cleaner.

arturpragacz commented:
Adding it to every single test is not something that looks like a sensible solution to me. Can you make it an autouse fixture instead?

epenet committed:
Ensure action have name and description translations

epenet committed:
flux ignore_missing_translations

epenet committed:
notify ignore_missing_translations

epenet committed:
onkyo ignore_missing_translations

epenet committed:
remember_the_milk ignore_missing_translations

epenet commented:
> There are some tests failing that need to be looked at. I think there are also merge conflicts. Can you please look into that? @epenet Cheers.

Rebased and cleaned up

epenet committed:
Simplify onkyo

epenet committed:
configurator ignore_missing_translations

epenet committed:
Simplify flux

frenck committed:
Merge branch 'dev' into epenet-20251208-1317

joostlek committed:
Merge branch 'dev' into epenet-20251208-1317

frenck committed:
Merge branch 'dev' into epenet-20251208-1317

joostlek reviewed (changes requested):
So the frontend test is now consistently failing on this PR for 3 times in a row. I think this might be related

epenet commented:
> So the frontend test is now consistently failing on this PR for 3 times in a row. I think this might be related

The weird thing is that on my machine the test is also failing on `dev`:

`pytest tests/components/frontend/test_storage.py::test_user_store_load_error`

epenet commented:
> > The weird thing is that on my machine the test is also failing on `dev`:
> > `pytest tests/components/frontend/test_storage.py::test_user_store_load_error`
> 
> Can confirm. Also fails on my machine.

Actually as spotted by @tr4nt0r the test itself passes... but throws the exception on teardown

I'm not really sure why we use a future in `async_user_store`

epenet committed:
Merge branch 'dev' into epenet-20251208-1317

piitaya commented:
I don't remember why but I changed from event to future in my previous PR : https://github.com/home-assistant/core/pull/157616/changes/425f73ddb35aafd76654c95c301e6fe1c4a5db74
May be event will work too...
